bgpd/bgp_bmp.c

```diff
@@ -801,11 +878,22 @@ static void bmp_eor(struct bmp *bmp, afi_t afi, safi_t safi, uint8_t flags)
 		if (!peer->afc_nego[afi][safi])
 			continue;
 
+		uint64_t peer_distinguisher = 0;
+		/* skip this message if peer distinguisher is not available */
+		if (bmp_get_peer_distinguisher(bmp, afi, peer_type_flag,
+					       &peer_distinguisher)) {
+			zlog_debug(
```

Comment:
Unguarded zlog_debug, please make sure we don't print debug messages without checking if debug is enalbed.